frontend/system/src/Admin/DrugEditForm/AddFieldButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { FlatButton } from '@common/components';
+import { PlusCircleIcon } from '@common/icons';
+import React from 'react';
+
+export const AddFieldButton = ({
+  label,
+  onClick,
+}: {
+  label: string;
+  onClick: () => void;
+}) => {
+  return (
+    <FlatButton
+      startIcon={<PlusCircleIcon />}
+      label={label}
+      onClick={onClick}
+      disableFocusRipple
+      sx={{
+        marginLeft: '20px',
+        '&.Mui-focusVisible': {
+          backgroundColor: '#e95c3029',
```

Comment:
The FlatButton has a strange bug where the focus ripple animation doesn't stop when you hit enter on the button, which takes the focus elsewhere on the page. 

Disabling the focus ripple and adding a background colour when the button is in focus appears to work reliably for both mouse and keyboard interactions